frontend/tests/App.spec.js

```diff
@@ -59,6 +60,73 @@ describe('App.vue', () => {
     expect(wrapper.vm.collections).toEqual("collection");
   })
 
+  it('changeItemsState - method', () => {
+    wrapper = initComponent(App);
+    const fqid1 = "extName1.extPublisher1.id1";
+    const item1 = {
+        id: "id1",
+        fqid: fqid1,
+        description: "description1",
+        title: "title1",
+        readState: "UNREAD",
+        labels: []
+    };
+    const fqid2 = "extName1.extPublisher1.id2";
+    const item2 = {
+        id: "id2",
+        fqid: fqid2,
+        description: "description2",
+        title: "title2",
+        labels: [],
+        items: [item1]
+    };
+    const collection1 = {
+        id: "id1",
+        title: "title1",
+        description: "description1",
+        itemIds: [],
+        type: 0,
+        items: [item2]
+    };
+    wrapper = initComponent(App);
+    wrapper.vm.showCollections([collection1]);
+    wrapper.vm.changeItemsState([{fqid:fqid1, readState: "READ"}]);
```

Comment:
You assert:
`expect(tarItem.readState).toEqual("READ");`

after calling `changeItemState`

I think you should also add an assertion about the `readState` **before** invoking `changeItemState`